apps/emqx/etc/emqx.conf

```diff
@@ -1023,15 +1023,31 @@ broker {
   ## Dispatch strategy for shared subscription
   ##
   ## @doc broker.shared_subscription_strategy
-  ## ValueType: random | round_robin | sticky | hash
+  ## ValueType: random | round_robin | sticky | hash | local
   ##   - random: dispatch the message to a random selected subscriber
   ##   - round_robin: select the subscribers in a round-robin manner
+  ##   - local: select the local subscriber otherwise random cluster-wise
   ##   - sticky: always use the last selected subscriber to dispatch,
   ##     until the subscriber disconnects.
   ##   - hash: select the subscribers by the hash of clientIds
   ## Default: round_robin
   shared_subscription_strategy = round_robin
 
+  ## Per-group dispatch strategy for shared subscription
+  ##
+  ## @doc broker.shared_subscription_group.$group_name.strategy
+  ## ValueType: random | round_robin | sticky | hash | local
+  ##   - random: dispatch the message to a random selected subscriber
+  ##   - round_robin: select the subscribers in a round-robin manner
+  ##   - local: select the local subscriber otherwise random cluster-wise
+  ##   - sticky: always use the last selected subscriber to dispatch,
+  ##     until the subscriber disconnects.
+  ##   - hash: select the subscribers by the hash of clientIds
+  ## Default: round_robin
+  shared_subscription_group {
+
```

Comment:
Would be nice to have commented out example configuration here.

Reply:
```suggestion

    ## example_group {
    ##   strategy = random
    ## }

```